clang/include/clang/Basic/DiagnosticLexKinds.td

```diff
@@ -932,6 +932,12 @@ def warn_module_conflict : Warning<
   InGroup<ModuleConflict>;
 
 // C++20 modules
+def err_module_decl_cannot_be_macros : Error<
+  "the name of a module%select{| partition}0 declaration cannot contains "
+  "an object-like macro %1, and the macro will not expand"
+  "%select{|; did you mean '%3'?}2">;
+def err_unxepected_paren_in_module_decl : Error<
+  "unexpected '(' after the name of a module%select{| partition}0 declaration">;
```

Comment:
"name of a module declaration" is a bit odd. I think "module name in a module declaration" is clearer.

```suggestion
def err_unxepected_paren_in_module_decl : Error<
  "unexpected '(' after the module name in a module%select{| partition}0 declaration">;
```